src/main/java/by/itechart/libmngmt/controller/command/commands/DeleteSearchedBookCommand.java

```
@@ -0,0 +1,60 @@
+package by.itechart.libmngmt.controller.command.commands;
+
+import by.itechart.libmngmt.controller.command.LibraryCommand;
+import by.itechart.libmngmt.entity.BookEntity;
+import by.itechart.libmngmt.service.BookService;
+import by.itechart.libmngmt.service.impl.BookServiceImpl;
+
+import javax.servlet.ServletException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class DeleteSearchedBookCommand extends LibraryCommand {
+    private BookService bookService = BookServiceImpl.getInstance();
+    private static DeleteSearchedBookCommand instance = new DeleteSearchedBookCommand();
+
+    public static DeleteSearchedBookCommand getInstance() {
+        return instance;
+    }
+
+    @Override
+    public void process() throws ServletException, IOException {
+        try {
+            Object[] booksIdsForDeleting = Arrays.stream(request.getParameterValues("bookid")).mapToInt(Integer::parseInt).boxed().toArray();
+            bookService.delete(booksIdsForDeleting);
+        } catch (Exception e) {
+
```

An empty catch, Add exception handling or logging here